src/test/resources/GenerateTest/buf_generate_fails_with_both_default_and_specified_buf_gen_template_files/build.gradle

```diff
@@ -2,6 +2,10 @@ plugins {
   id 'build.buf'
 }
 
+repositories {
```

Comment:
Why is this necessary now (and seemingly only for some tests)? If it is necessary, then I think it should be added to the example code (or at least a blurb addressing the error that occurs if it is absent) in the readme as well.

Reply:
If I remember correctly, it's because of reordering steps in which they're executed. These tests test some kind of failures. Originally, these failures happened before buf executable was resolved. After the change, buf executable is resolved before this failure occurs, so the test fails on something different. Providing the repo makes the test reach the desired failure.